indexer-app/src/cmr/indexer/data/elasticsearch.clj

```diff
@@ -59,6 +59,10 @@
         (:transaction-id concept)
         (map :transaction-id (:service-associations concept))))
 
+(defmethod get-elastic-version :subscription
```

Comment:
why does this have to be `transaction-id`?

Reply:
Because if you look at the service and variable, they pick the max of transaction-id and the transaction-ids of the associations. so it makes sense to pick the transaction-id when you don't have associations.

Reply:
Subscription concept is not associated with any concepts. It makes more sense to just delete this function and let the default implementation (which is for any concepts without associations) to handle the elastic version.

Reply:
This is not the one being used right now(see my other comment about the same name defmethod ) The default one over there is using revision-id, which I was using before but it didn't work. Remember I was able to search for deleted concept at one point?  That's because when the concept is deleted, the delete-document didn't work in elastic search. the revision-id is lower than the current id in ES, which was using transaction.

Reply:
I don't follow what you are saying.